scripts/v8-to-v9.py

```
@@ -0,0 +1,157 @@
+    unjailed = []
+    for state in blank_data['app_state']['operator']['opt_states']:
+        if state['opt_info']['jailed'] == True:
+            # convert the jailed operator to unjailed
+            state['opt_info']['jailed'] = False
+            addr = state['key'].split("/")[0]
+            unjailed.append(addr)
+        # set an empty value for the new field.
+        state['opt_info']['jail_toggle_heights'] = []
+    for operator in unjailed:
+        # find the usd value
+        obj = [ o for o in blank_data['app_state']['operator']['operator_usd_values'] if o['key'] == f"{dogfood_addr}/{operator}" ]
```

Flake8 (7.2.0)

[error] whitespace after '['

(E201)